basiclti/basiclti-tool/src/java/org/sakaiproject/blti/tool/LTIAdminTool.java

```diff
@@ -162,6 +194,152 @@ public String buildMainPanelContext(VelocityPortlet portlet, Context context,
 		return buildToolSystemPanelContext(portlet, context, data, state);
 	}
 	
+	/**
+	 * Filter action : allows to filter tool site list with the given tool_id
+	 * 
+	 * Accepted parameters : id
+	 * @param data
+	 */
+	public void doFilter(RunData data) {
```

Comment:
Doesn't match the code style of this class. Opening braces of new methods should be on their own line.